napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -276,6 +276,10 @@ def _on_size_change(self):
         """Receive layer model size change event and update point size slider."""
         with self.layer.events.size.blocker():
             value = self.layer.current_size
+            if value < self.sizeSlider.minimum():
+                self.sizeSlider.setMinimum(int(value - 1))
```

Comment:
I'm not happy with the negative size of points. There should be a guardian that will prevent this.

Reply:
I put a minimum value of 1 for this case here.
Additionally it is now forbidden to have negative values in the current_size value at all.
I think that this makes sense?